packages/analyticsdx-template-lint/src/schemas/readiness-schema.json

```diff
@@ -34,7 +34,8 @@
               "${Readiness.sobjectCount > Variables.minimumCount}",
               "${Readiness.datasetRowCount >= Constants.minNumRows}",
               "${Readiness.apexResult.name == 'requiredName'}"
-            ]
+            ],
+            "defaultSnippets": [{ "label": "\"\"", "body": "${0}" }]
```

Comment:
If you have `examples` on a field, vscode won't include the default code-completion (e.g. `""` for a string type like this one). Doing the `defaultSnippet` will.